src/evergreen/api.py

```diff
@@ -50,6 +51,8 @@
 START_WAIT_TIME_SEC = 2
 MAX_WAIT_TIME_SEC = 5
 
+Requester = Enum('Requester', 'patch_request gitter_request github_pull_request merge_test ad_hoc')
```

Comment:
[nit]/[question] Are we using lowercase across the board? I think uppercase is more "standard" (signifying constant value, parity with other languages), and we could call `requester.name.lower()` below.

Perhaps worth making a dictionary keyed by values of this enum? In general we don't know that a web API will accept parameters that are conveniently similar to python syntax, and we would need to do (e.g. when API uses camelCase):
```
OPTIONS = {
    Requester.PATCH_REQUEST: 'patchRequest'
}
...
params = {
    'requester': OPTIONS[requester]
}
```

Reply:
updated.